packages/utils/src/logger/index.ts

```diff
@@ -0,0 +1,35 @@
+import debug, { Debugger } from "debug";
+
+const APP_NAME = "js-waku";
+
+export class Logger {
+  private _info: Debugger;
+  private _warn: Debugger;
+  private _error: Debugger;
+
+  static createDebugNamespace(level: string, prefix?: string): string {
+    return prefix ? `${APP_NAME}:${level}:${prefix}` : `${APP_NAME}:${level}`;
+  }
+
+  constructor(prefix?: string) {
+    this._info = debug(Logger.createDebugNamespace("INFO", prefix));
+    this._warn = debug(Logger.createDebugNamespace("WARN", prefix));
+    this._error = debug(Logger.createDebugNamespace("ERROR", prefix));
+
+    this._info.log = console.info.bind(console);
```

Comment:
Nothing is better than good old `console.*` 😄

Reply:
is this compatible with libp2p logging approach?